src/error.rs

```diff
@@ -43,13 +50,15 @@ impl error::Error for SdpParserError {
             SdpParserError::Unsupported { ref message, .. } |
             SdpParserError::Sequence { ref message, .. } => message,
             SdpParserError::Integer(ref err) => error::Error::description(err),
+            SdpParserError::Address(ref err) => error::Error::description(err),
```

Comment:
On the line above I actually can't do it, because there is some ambiguity so the compiler doesn't know which function to call.
I actually had initially this one as `err.description()`, but then convert it so it's consistent with the line above it.